src/components/contributions/admin/adminContributionsFilterVM.js

```diff
@@ -9,3 +9,16 @@ var vm = adminApp.AdminContributionsFilter.VM = m.postgrest.filtersVM({
 // Set default values
 vm.state('');
 vm.gateway('Pagarme');
+
+var momentFromString = function(date){
```

Comment:
What you think of making this function global? It actually looks like a momentify reverse function. We would just need to add a format parameter.